pkg/skaffold/debug/transform.go

```diff
@@ -311,7 +311,8 @@ func isPortAvailable(podSpec *v1.PodSpec, port int32) bool {
 // Returns a debugging configuration description with associated language runtime support
 // container image, or an error if the rewrite was unsuccessful.
 func transformContainer(container *v1.Container, config imageConfiguration, portAlloc portAllocator) (ContainerDebugConfiguration, string, error) {
-	// update image configuration values with those set in the k8s manifest
+	// Update the image configuration's environment with those set in the k8s manifest.
+	// (Environment variables in the k8s container's `env` add to the image configuration's `env` settings rather than replace.)
```

Comment:
I am not sure i understand this comment. 
The only two env variables replaced or added  are `NODE_OPTIONS` and `PATH` right?

Reply:
This was a general comment: the k8s `command`, `arguments`, and `workingDir` attributes replace the image's `entrypoint`, `cmd`, and `workdir` values (respectively).  I clearly mistakenly thought the environment variables replaced the image's values too (the [API docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core) are not clear).